src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/ConnectionTestParametersData.cs

```diff
@@ -0,0 +1,336 @@
+                    HostNameInCertificate = null,
+                    TestResult = true
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.On,
+                    Encrypt = SqlConnectionEncryptOption.Mandatory,
+                    TrustServerCertificate = true,
+                    Certificate = null,
+                    HostNameInCertificate = null,
+                    TestResult = true
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.On,
+                    Encrypt = SqlConnectionEncryptOption.Mandatory,
+                    TrustServerCertificate = false,
+                    Certificate = s_fullPathToCer,
+                    HostNameInCertificate = null,
+                    TestResult = true
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.On,
+                    Encrypt = SqlConnectionEncryptOption.Mandatory,
+                    TrustServerCertificate = true,
+                    Certificate = s_fullPathToCer,
+                    HostNameInCertificate = null,
+                    TestResult = true
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.On,
+                    Encrypt = SqlConnectionEncryptOption.Mandatory,
+                    TrustServerCertificate = false,
+                    Certificate = null,
+                    HostNameInCertificate = s_hostName,
+                    TestResult = false
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.On,
+                    Encrypt = SqlConnectionEncryptOption.Mandatory,
+                    TrustServerCertificate = true,
+                    Certificate = null,
+                    HostNameInCertificate = s_hostName,
+                    TestResult = true
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.Required,
+                    Encrypt = SqlConnectionEncryptOption.Optional,
+                    TrustServerCertificate = false,
+                    Certificate = null,
+                    HostNameInCertificate = null,
+                    TestResult = false
+                },
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.Required,
+                    Encrypt = SqlConnectionEncryptOption.Mandatory,
+                    TrustServerCertificate = false,
+                    Certificate = null,
+                    HostNameInCertificate = null,
```

Comment:
same here

Reply:
Sorry could you clarify what same here means?

Reply:
https://github.com/dotnet/SqlClient/pull/2439#discussion_r1571201554https://github.com/dotnet/SqlClient/pull/2439#discussion_r1571201554

Reply:
"same here" I guess has been addressed.